src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react'
+
+interface ProfileProps 
+{
+  name: string,
+  email: string,
+  userName: string,
+  emergencyName: string,
+  emergencyNumber: string 
+}
```

Comment:
Curious about why this doesn't throw errors when we don't include one of these props but it seems to work syntactically. Great work!